configs/ReflexCtrlNES/BoardConfig.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: Copyright (c) 2021 Jason Skuby (mytechtoybox.com)
+ */
+
+#ifndef PICO_BOARD_CONFIG_H_
+#define PICO_BOARD_CONFIG_H_
+
+#include "enums.pb.h"
+#include "class/hid/hid.h"
+
+#define BOARD_CONFIG_LABEL "Reflex CTRL Genesis 6"
+
+// Input Mode additions section.
```

Comment:
To keep with the cleanup efforts in board-configs previously, all of the // lines except the top Licence and FireCopyrightText ones can be removed.